docs/web3.eth.rst

```diff
@@ -1108,6 +1100,38 @@ The following methods are available on the ``web3.eth`` namespace.
     ``(web3, transaction_params)`` and return a gas price denominated in wei.
 
 
+.. py:method:: Eth.subscribe(subscription_identifier, subscription_params)
+
+      * Delegates to ``eth_subscribe`` RPC Method
+
+      Only available on persistent connection providers: ``WebSocketProvider`` and ``AsyncIPCProvider``.
+
+      Returns a subscription ID that can be used to track or unsubscribe from an event's logs.
```

Comment:
I'm not sure about the use of "logs" here? I think it can make sense but it may muddy with ethereum logs. Does just event make sense?

nit: track a subscription to / unsubscribe from ? maybe ?

```suggestion
      Returns a subscription ID that can be used to track a particular subscription to, or unsubscribe from, an event.
```